core/domain/suggestion_registry.py

```diff
@@ -1095,3 +1095,98 @@ def reject(self, reviewer_id, rejection_message):
         SuggestionTranslateContent),
     suggestion_models.SUGGESTION_TYPE_ADD_QUESTION: SuggestionAddQuestion
 }
+
+
+class ReviewerAndSuggestionCounts(python_utils.OBJECT):
+    """Domain object for the ReviewerAndSuggestionCountsModel.
+
+    Attributes:
+        translation_reviewer_counts_by_lang: dict. A dictionary where the keys
+            represent the languages that translation suggestions are offered in
+            and the values correspond to the total number of reviewers who have
+            permission to review translation suggestions in that language.
+        translation_suggestion_counts_by_lang: dict. A dictionary where the
+            keys represent the languages that translation suggestions are
+            offered in and the values correspond to the total number of
+            translation suggestions that are currently in review in that
+            language.
+        question_reviewer_count: int. The total number of reviewers who have
+            permission to review question suggestions.
+        question_suggestion_count: int. The total number of question
+            suggestions that are currently in review.
+    """
+
+    def __init__(
+            self, translation_reviewer_counts_by_lang,
+            translation_suggestion_counts_by_lang,
+            question_reviewer_count, question_suggestion_count):
+        self.translation_reviewer_counts_by_lang = (
+            translation_reviewer_counts_by_lang
+        )
+        self.translation_suggestion_counts_by_lang = (
+            translation_suggestion_counts_by_lang
+        )
+        self.question_reviewer_count = question_reviewer_count
+        self.question_suggestion_count = question_suggestion_count
+
+    def validate(self):
+        """Validates the ReviewerAndSuggestionCounts object.
+
+        Raises:
+            ValidationError. One or more attributes of the
+                ReviewerAndSuggestionCounts object is invalid.
+        """
+        for language_code, reviewer_count in (
+                self.translation_reviewer_counts_by_lang.items()):
```

Comment:
Is it also worth verifying that valid language codes are present in the dictionary? Ditto for the dictionary below.

Also, probably worth adding tests for these cases.